src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -146,9 +147,16 @@ public AddressBook load() throws StorageOperationException {
 
         // create empty file if not found
         } catch (FileNotFoundException fnfe) {
-            final AddressBook empty = new AddressBook();
-            save(empty);
-            return empty;
+            final AddressBook emptyAddressBook = new AddressBook();
+            
+            try {
+                createEmptyFile(path);
+            } catch (IOException ioe) {
+                throw new StorageOperationException("Error writing to file: " + path);
+            }
+            
```

Comment:
indentations are way off. I remember commenting about coding style adherence in your PR last week too. This is easy to do with eclipse auto format. Please follow.